packages/block/package.json

```diff
@@ -49,7 +49,7 @@
   "dependencies": {
     "@ethereumjs/common": "^4.4.0",
     "@ethereumjs/rlp": "^5.0.2",
-    "@ethereumjs/trie": "^6.2.1",
+    "@ethereumjs/mpt": "^6.2.1",
```

Comment:
Not sure if this is clean in the PR, do we want this rename + keep the same version here? :thinking: (I realize that this will likely mean we need a release of the new package though to pass CI - or will this be handled correctly?)

Reply:
I thought the same, seemed weird not to update the version at all